br/pkg/stream/table_mapping.go

```diff
@@ -293,11 +293,29 @@ func (tm *TableMappingManager) MergeBaseDBReplace(baseMap map[UpstreamID]*DBRepl
 			existingDBReplace.DbID = newID
 		}
 
+		// db replace in `TableMappingManager` has no name yet, it is determined by baseMap.
+		// TODO: update the name of the db replace that is not exists in baseMap. Now it is OK because user tables' name is not used.
```

Comment:
```suggestion
		// TODO: update the name of the db replace that is not exists in baseMap.
		// Now it is OK because user tables' name is not used.
```